runner/execer/os/execer.go

```diff
@@ -212,7 +232,7 @@ func (e *execer) monitorMem(p *process, memCh chan scootexecer.ProcessStatus) {
 							"tag":    p.Tag,
 							"jobID":  p.JobID,
 							"taskID": p.TaskID,
-						}).Tracef("ps after increasing mem_cap utilization for pid %d", pid)
+						}).Tracef("ps after increased memory utilization for pid %d", pid)
```

Comment:
not really related, but do we use trace logs anywhere? i think staging/devel use debug.

Reply:
yeah, don't think so. Will update this to debug.